encodings/fastpfor/js/package.json

```diff
@@ -19,7 +19,7 @@
     "test:lint": "eslint src --ext .ts",
     "test:prettier": "prettier \"src/**/*.ts\" --list-different",
     "test:spelling": "cspell \"{README.md,.github/*.md,src/**/*.ts}\"",
-    "test:unit": "nyc --silent ava",
+    "test:unit": "nyc --silent ava src/tests/unit/**/*.ts",
```

Comment:
if we are not running everything here, we should probably add `test:bench` and maybe some other `test:...` to run the other types. Otherwise they will be in the code, but we won't have an easy way to run them.

Reply:
Agreed that this should be added, but previous to this PR these benchmarks were not running and were breaking the `test:unit` target since they don't use `ava`. So I was considering this as a first step to get the unit tests running.